### PR TITLE
bext: make code intel popup buttons smaller on GitHub

### PR DESCRIPTION
**Pull request by taras-yemets**
Closes [#31359](https://github.com/sourcegraph/sourcegraph/issues/31359)

Screenshot from 2022-02-23 13-50-25

## Test plan

See steps to reproduce in the original issue

### CODE REVIEW

File: client/browser/src/shared/code-hosts/github/codeHost.module.scss [hunk 1]
```diff
@@ -51,6 +51,12 @@
     margin-bottom: 4px;
 }
 
+.hover-overlay {
+    /* GitHub buttons are larger than buttons on other codehosts and Sourcegraph cloud,
+     * so we make the container wider to prevent content overflow. */
+    max-width: 38rem !important;
```
Comment (taras-yemets) [inline client/browser/src/shared/code-hosts/github/codeHost.module.scss:57]: We increase the default container width defined [client/shared/src/hover/HoverOverlay.module.scss](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/shared/src/hover/HoverOverlay.module.scss?L15).

File: client/browser/src/shared/code-hosts/github/codeHost.tsx [hunk 2]
```diff
@@ -722,7 +722,7 @@ export const githubCodeHost: GithubCodeHost = {
         actionItemIconClass: classNames(styles.icon, 'v-align-text-bottom'),
     },
     hoverOverlayClassProps: {
-        className: 'Box',
+        className: classNames('Box', styles.hoverOverlay),
         actionItemClassName: 'btn btn-secondary',
```
Comment (erzhtor) [inline client/browser/src/shared/code-hosts/github/codeHost.tsx]: Maybe we could add `btn-sm` class.

**Review by erzhtor — APPROVED** [review]
👍

**Comment by jjinnii** [comment]
So fast @taras-yemets ! 🚀

**Pull request merged**